uint8 header_byte
uint8 right_v_byte
uint8 right_h_byte
uint8 left_v_byte
uint8 left_h_byte
uint8 button_byte
uint8 end_byte
uint8 checksum_byte
